packages/offix-client/src/links/LinksBuilder.ts

```diff
@@ -81,21 +81,22 @@ export const defaultHttpLinks = async (config: OffixClientConfig, offlineLink: A
   links.push(conflictLink);
   const retryLink = ApolloLink.split(OfflineMutationsHandler.isMarkedOffline, new RetryLink(config.retryOptions));
   links.push(retryLink);
-
+  
   if (config.authContextProvider) {
     links.push(createAuthLink(config));
   }
   const localFilterLink = new LocalDirectiveFilterLink();
   links.push(localFilterLink);
 
+  const httpLinkOptions = {
+    uri: config.httpUrl,
+    ...httpOptions
```

Comment:
bug here. httpOptions also has a `uri` key.

Reply:
Yeah.. Moving this to config will mean that we will have 4 places that will modify behaviour of the entire client significantly. Documenting order and priority here may be challenging. 

- config.httpOptions - contains full scope of options
- config.url = specifies url to be used.
- config.fileUpload = true (use upload link instead of HttpLink - compatibility)
- config.wsURL (use websockets on top of http)